src/main/java/org/usfirst/frc4048/common/util/SwerveModuleProfileV2.java

```diff
@@ -0,0 +1,74 @@
+package org.usfirst.frc4048.common.util;
+
+
+public enum SwerveModuleProfileV2 {
+    MK4(true, false, true, false,
+            12.8, 6.75, false),
+    MK4I(true, false, true, false,
+            150f / 7f, 8.142857, true); //old
+
+    SwerveModuleProfileV2(boolean frontRightInverted, boolean frontLeftInverted, boolean backRightInverted, boolean backLeftInverted, double steerRatio, double driveRatio, boolean steerInverted) {
```

Comment:
Why do we need both `SwerveModuleProfile` and `V2`?

Reply:
The old non advantage kit swerveV2 relies on `SwerveModuleProfile`. Since swerveV3 introduced changes to this class I made another version so they would both still work. (the better solution would be to update swerveV2)